libs/island-ui/storybook/config/main.js

```diff
@@ -1,11 +1,74 @@
+const path = require('path')
+const TreatPlugin = require('treat/webpack-plugin')
+
+const rootDir = (dir) => path.resolve(__dirname, dir)
+
 module.exports = {
-  stories: ['../../core/src/**/*.stories.@(tsx|mdx)'],
+  stories: [
+    '../../core/src/**/*.stories.@(tsx|mdx)',
+    '../../../application/ui-fields/src/lib/AsGuide.stories.mdx',
+    '../../../application/ui-fields/**/*.stories.@(tsx|mdx)',
+  ],
   addons: [
-    '@storybook/addon-a11y/register',
-    '@storybook/addon-docs',
+    '@storybook/addon-a11y',
     '@storybook/addon-essentials',
-    '@storybook/addon-controls',
     'storybook-addon-designs',
-    './title/register',
   ],
+  webpackFinal: (config) => {
+    config.plugins.push(new TreatPlugin())
+    config.resolve.extensions.push('.tsx', '.ts')
```

Comment:
Doesn't storybook support tsx?

Reply:
It might do, I will try. I just move the config but didn't try removing this bit.